postfixadmin/stack.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: notSecureChangeMe
+      MYSQL_DATABASE: postfixadmin
+      MYSQL_USER: postfixadmin
+      MYSQL_PASSWORD: example
+
+  postfixadmin:
+    depends_on:
+      - db
```

Comment:
FYI, `depends_on:` doesn't really do much these days (especially with services in a stack), and I'd recommend against relying on it (or using it).